routes/api/ordered-dish.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const router = express.Router();
+const db = require("../../models/index");
+const OrderedDish = db.orderedDish;
+
+router.get('/all', (req, res) => {
+    OrderedDish.findAll()
+        .then(data => res.send(data))
+        .catch((err) => {
+            console.log(err);
+            res.status(500).send('Failed to get all');
+        });
+});
+
+router.get('/', (req, res) => {
+    OrderedDish.findByPk(req.query.id)
+        .then(data => res.send(data))
+        .catch((err) => {
+            console.log(err);
+            res.status(500).send('Failed to get');
+        });
+});
+
+router.post('/', (req, res) => {
+    OrderedDish.create(req.body)
```

Comment:
Myślę że tu powinno być że w req.body powinno być id albo nazwa elementu z menu i tu powinna być zrobiona transakcja bo tak to srednio wykorzystamy relacje